src/ci/resources/tes_application.conf

```diff
@@ -7,7 +7,17 @@ filesystems {
     global {
       class = "cromwell.filesystems.blob.BlobFileSystemConfig"
       config {
-        workspace-manager-url: "https://workspace.dsde-dev.broadinstitute.org"
+        container: "cromwell"
+        endpoint: "https://<storage-account>.blob.core.windows.net"
+        subscription: "00001111-2222-3333-aaaa-bbbbccccdddd"
+        # WSM config is needed for accessing WSM-managed blob containers
+        # created in Terra workspaces.
+        workspace-manager {
+          url: "https://workspace.dsde-dev.broadinstitute.org"
+          workspace-id: "00001111-2222-3333-aaaa-bbbbccccdddd"
+          container-resource-id: "00001111-2222-3333-aaaa-bbbbccccdddd"
+          overrideB2cToken: "Zardoz"
```

Comment:
This naming is not consistent with what the code is expecting (which makes me realize we don't have test coverage for parsing this option).